### PR TITLE
Feat: Added Modal component

### PR DESCRIPTION
**Pull request by shuntzhere**

### CODE REVIEW

**Review by sadathshariff — APPROVED** [review]
The code looks clean and good

File: components/grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+          <div class="left-nav flex flex-items-center justify-between">
+            <img class="hamburger" src="/assets/svg/hamburger-solid.svg" />
+            <a href="../../../index.html">
+              <img class="logo flex-grow" src="/assets/svg/logo.svg" />
+            </a>
+            <a
+              class="right-nav flex"
+              href="https://github.com/shuntzhere/Shore-UI"
+              >Github</a
+            >
```
Comment (anushasunil) [inline components/grid/grid.html:23]: minor refactoring needed in code format

**Review by anushasunil — APPROVED** [review]
The code looks neat and the UI is awesome. Great work 👍
Just a small suggestion below

File: components/grid/grid.html (hunk 1 above)
Comment (shuntzhere) [reply components/grid/grid.html:23]: Yes, I'll do that.

**Pull request merged**